### PR TITLE
Move to a newer version of library-go and external snapshotter

### PR DESCRIPTION
**Pull request by huffmanca**
Updates to use a newer version of library-go, client-go, and the external snapshot controller.

### CODE REVIEW

**Comment by huffmanca** [comment]
I attempted to update the external-snapshotter, which ended up pulling in other dependencies which has broken this for now. I'm continuing to examine this.

File: pkg/operator/operator_test.go [hunk 1]
```diff
@@ -256,7 +260,7 @@ func getDeployment(args []string, image string, modifiers ...deploymentModifier)
 	if dep.Annotations == nil {
 		dep.Annotations = map[string]string{}
 	}
-	dep.Annotations["operator.openshift.io/pull-spec"] = image
+	addDeploymentHash(dep)
```
Comment (jsafrane) [inline pkg/operator/operator_test.go:263]: Can you use helper func from library-go? https://github.com/openshift/library-go/blob/master/pkg/operator/resource/resourceapply/apps.go#L59

Comment (huffmanca) [reply pkg/operator/operator_test.go]: This has been updated.

File: pkg/dependencymagnet/dependencymagnet.go [hunk 2]
```diff
@@ -5,6 +5,7 @@
 package dependencymagnet
 
 import (
-	_ "github.com/jteeuwen/go-bindata/go-bindata"
-	_ "github.com/openshift/library-go/alpha-build-machinery"
+	_ "github.com/go-bindata/go-bindata"
+	_ "github.com/go-bindata/go-bindata/go-bindata"
```
Comment (bertinatto) [inline pkg/dependencymagnet/dependencymagnet.go:9]: There seems to be an additional go-bindata entry here

Comment (huffmanca) [reply pkg/dependencymagnet/dependencymagnet.go:9]: When I was testing this it needed both `github.com/go-bindata/go-bindata` and `github.com/go-bindata/go-bindata/go-bindata` to have both `make` and `make update` complete successfully.

I'm not certain why, as I'd assume the longer path would be sufficient.

**Pull request merged**